custom-vu/src/main/kotlin/jces1209/vu/utils/boards/BoardsFrequencyManager.kt

```diff
@@ -0,0 +1,41 @@
+package jces1209.vu.utils.boards
+
+import java.util.*
+
+class BoardsFrequencyManager {
+
+    private val csvPropertyFile = "BoardUsageFrequency.csv"
```

Comment:
Please add a comment with the description regarding this file: `location`, `template`, etc
and add a TODO to get the file path from the tenant properties